src/client/components/MapHeader.tsx

```diff
@@ -77,7 +77,7 @@ const style: ThemeUIStyleObject = {
     borderColor: "gray.2",
     padding: "5px",
     top: "100px",
-    zIndex: 99999,
+    zIndex: 999,
```

Comment:
This is still too large - in general, try and keep your `z-index` values as low as you can afford to; in this case just a `z-index` of `1` was sufficient to get this to appear above the map

![image](https://user-images.githubusercontent.com/4432106/127669379-7fb46fc5-1ec4-4923-8cb7-c1d774ba95c1.png)


```suggestion
    zIndex: 1,
```